tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -60,6 +60,45 @@ void throw_helper(const charT* fmt, const Args&... vals) {
     }
 }
 
+template <class back_inserter>
+struct move_only_back_inserter {

```

Comment:
this may be simpler if it just always uses back_insert_iterator and is parameterized on it's type parameters (or just charT)